Test failed, but not sure how its related.
```
FAIL: test_case_pillow_indicators (mvp_docs.tests.test_pillows.IndicatorPillowTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "custom/_legacy/mvp_docs/tests/test_pillows.py", line 115, in test_case_pillow_indicators
    self.assertNotEqual(indicator_case.computed_, {})
AssertionError: {} == {}
```

--- a/corehq/apps/indicators/utils.py
+++ b/corehq/apps/indicators/utils.py
@@ -1,10 +1,10 @@
 from couchdbkit import ResourceNotFound
 from django.conf import settings
-from corehq.util.quickcache import quickcache
+from corehq.util.quickcache import skippable_quickcache
 from dimagi.utils.couch.database import get_db
 
 
-@quickcache([], timeout=60)
+@skippable_quickcache([], lambda: settings.UNIT_TESTING, timeout=60)
 def get_indicator_config():
     try:
         doc = get_db().open_doc('INDICATOR_CONFIGURATION')

